README.adoc

```diff
@@ -276,10 +329,35 @@ party service, you should separate your `GET`/`POST` requests from your data con
 
 The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
 
+// static guide instructions:
+ifndef::cloud-hosted[]
 You can find your service at http://localhost:9080/artists[http://localhost:9080/artists^].
 
 Now, you can access the endpoint at http://localhost:9080/artists/total[http://localhost:9080/artists/total^] to see the total number of artists,
 and you can access the endpoint at `\http://localhost:9080/artists/total/<artist>` to see a particular artist's total number of albums. 
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+You can find your service at **http://localhost:9080/artists** by running the following curl command:
```

Comment:
You can find your service at the **http://localhost:9080/artists** endpoint by running the following curl command